docker/packer/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM demisto/python3-deb:3.8.6.12176
+
+RUN apt-get update
+
+RUN apt-get install curl jq unzip openssh-client -y
+
+RUN curl -so packer.zip $(curl -sk https://www.packer.io/downloads.html |grep -oE https://releases\.hashicorp\.com/packer/\[0-9.\]\{5\}/packer_\[0-9.\]\{5\}_linux_amd64\.zip)
```

Comment:
Please use a specific version which is locked.

Reply:
Or if you want the latest include a printout of the version when done installing as we do with gcloud install.